src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeDeletionTest.java

```diff
@@ -36,6 +38,8 @@
 
     private transient CloudBlobContainer testContainer;
 
+    private List<String> zipFilesToDelete = new ArrayList<>();
```

Comment:
a comment (or different name) would be good, to clarify that this is for the cleanup after tests only.

Reply:
Will do